xml/seccheck.xml

```diff
@@ -172,19 +172,20 @@
         </entry>
         <entry>
          <para>
-          checks for mail aliases which execute programs
+          checks for mail aliases that execute programs
          </para>
         </entry>
        </row>
-       <row>
+       <row> <!-- TODO surely we do not support rhosts anymore- cs, 
+         June 2021-->
         <entry>
          <para>
           <filename>.rhosts</filename> check
          </para>
         </entry>
         <entry>
          <para>
-          checks if users' <filename>.rhosts</filename> file contain + signs
+          checks if users' <filename>.rhosts</filename> files contain + signs
```

Comment:
mark + with tag or quotation marks or so? I had to read the sentence twice to understand

Reply:
Fair. I have wrapped it in [literal]